### PR TITLE
DDF-1413 Modified unit test for robustness

### PR DESCRIPTION
**Pull request by kguerriero**
[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/157)

### CODE REVIEW

**Comment by coyotesqrl** [comment]
lgtm

**Comment by kguerriero** [comment]
@kcwire @lessarderic @coyotesqrl

File: catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrProvider.java [hunk 1]
```diff
@@ -1314,10 +1314,11 @@ private void addAndDeleteMetacards(int metacardCount)
         }
 
         DeleteResponse deleteResponse = delete(ids.toArray(new String[metacardCount]));
-        assertThat(deleteResponse.getDeletedMetacards().size(), is(metacards.size()));
+        List<Metacard> deletedMetacards = deleteResponse.getDeletedMetacards();
+        assertThat(deletedMetacards.size(), is(metacards.size()));
 
         for (int i = 0; i < metacardCount; i++) {
-            assertThat(deleteResponse.getDeletedMetacards().get(i).getId(), is(ids.get(i)));
+            assertThat(ids.contains(deletedMetacards.get(i).getId()), is(true));
```
Comment (lessarderic) [inline catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrProvider.java]: You should be able to do `assertThat(ids, contains(deletedMetacards.get(i).getId());`

**Comment by lessarderic** [comment]
:+1:

**Comment by kcwire** [comment]
Looks good with Eric's recommendation.

**Comment by lessarderic** [comment]
For some reason, the change I suggested doesn't seem to work. We'll keep looking at it after Ken's demo but in the mean time I'll hero Ken's original fix so we're ready to merge that change if needed.

**Pull request closed without merging**